install/check-minimum-requirements.sh

```diff
@@ -3,54 +3,68 @@ echo "${_group}Checking minimum requirements ..."
 source "$(dirname $0)/_min-requirements.sh"
 
 # Check the version of $1 is greater than or equal to $2 using sort. Note: versions must be stripped of "v"
-function vergte () { printf "%s\n%s" $1 $2 | sort --version-sort --check=quiet --reverse; echo $?; }
-
-
-
-DOCKER_VERSION=$(docker version --format '{{.Server.Version}}')
```

Comment:
I decided to split these into multiple functions, I thought that would be clearer.